diagramming-tool/package.json

```diff
@@ -28,7 +26,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom --testMatch='**/App.test.js'",
+    "test": "react-scripts test --env=jsdom --testMatch=**/Component/Dashboard/Canvas.test.jsx --coverage",
```

Comment:
Just mention the general test command instead of particular file command

Reply:
You can change this now or in the next PR just let me know... If in next PR i can approve this